README.rst

```diff
@@ -34,8 +35,72 @@ To install from source:
 
 .. installation-end-marker
 
-Next Steps
-----------
+Getting Started
+===============
+
+Sign up for the Leap quantum cloud service here:
+`Leap signup <https://cloud.dwavesys.com/leap/signup>`_. Leap provides a
+browser-based integrated development environment (IDE) with Ocean preinstalled.
+
+Start learning with the following D-Wave resources:
+
+* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_ to
+  learn about quantum computers and how to use them.
+
+* `Getting Started with Ocean <https://docs.ocean.dwavesys.com/en/stable/getting_started>`_
+  to install and start coding with Ocean software.
+
+* `dwave-examples <https://github.com/dwave-examples>`_ for code examples
+  and Jupyter Notebooks.
+
+* `D-Wave website <https://www.dwavesys.com/learn/resource-library>`_ for
+  whitepapers and additional resources.
+
+Example Quantum Program
+-----------------------
+
+The following lines of code solve and visualize a
+`random <https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generators.html>`_
+problem on a quantum computer.
+
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+>>> from dimod.generators import ran_r
+>>> import dwave.inspector
+...
+>>> bqm = ran_r(1, 30)
+>>> sampler = EmbeddingComposite(DWaveSampler())
+>>> sampleset = sampler.sample(bqm, num_reads=100)
+>>> dwave.inspector.show(sampleset)
+
+The left side of the visualized solution represents the problem's variables as
+circles, with white dots for variables assigned values of -1 and blue dots for
+values of +1; the colors of the connecting lines represent values of the
+quadratic coefficients for each pair of variables. The right side shows the qubits
+representing these variables on a quantum processing unit.
+
+.. image:: docs/_static/inspector_bqm_ran_r_30.png
+
+Support
+=======
+
+Find support here:
+
+* `Leap user community <https://support.dwavesys.com/hc/en-us/community/topics>`_
+  to converse with a large community of D-Wave users.
+* `Leap help center <https://support.dwavesys.com/hc/en-us>`_
+  to search the Leap knowledge base.
+* `GitHub issues <https://github.com/dwavesystems>`_ to open issues or request
```

Comment:
Maybe link to the sdk itself or https://github.com/dwavesystems/dwave-ocean-sdk/issues

Reply:
I was thinking that the higher-level link would let users open an issue in the relevant package repo, at least for now.

Reply:
If I was a kind of user that needs guidance on how to file Ocean issues, I would be lost with this link.

IMO, we should either link as Alex suggests, or link to `dwavesystems`, but change wording.

Reply:
Changed to 
```
`SDK GitHub repo <https://github.com/dwavesystems/dwave-ocean-sdk/issues>`_ to
open issues or request features on the Ocean SDK or on any one of its
`packages <https://github.com/dwavesystems>`_.
```